db/data.js

```diff
@@ -605,9 +598,9 @@ const produce = [
   {
     foodName: "Guinea Fowl",
     imageURL:
-      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637899/Seasonal/057-organic-food_pikcaf.png",
+      "https://res.cloudinary.com/dvpop7e9w/image/upload/v1666175320/Seasonal/016-fowl_ac08xk.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "A small domesticated bird, which is a little bit gamier in taste than chicken. Guinea fowl originate from Africa, though can be found across Europe. They are at their best in the autumn months, between September and November. Guinea fowl has a lower fat content and a higher fat content than chicken. As guinea fowl is drier than chicken naturally, it can dry out while cooking, so take precautions not to overcook the bird. Wrapping guinea fowl with pancetta can prevent it from drying out, as can marinating it. Guinea fowl is a great option for roasting with stuffing.",
```

Comment:
"Guinea fowl has a lower fat content and a higher fat content than chicken."
Both lower and higher fat content!? Incredible!